src/Form/Extension/DependencyInjectionExtension.php

```diff
@@ -20,6 +20,8 @@
 /**
  * This proxy class help to keep BC code with < SF2.8 form behavior by restoring
  * the type as a code and not as a class.
+ *
+ * @deprecated use FQCNs instead
```

Comment:
`deprecated since...` please follow the format described on the contributing guide.